docs/index.asciidoc

```diff
@@ -19,6 +19,8 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 // include::{include_path}/plugin_header.asciidoc[]
 
+experimental[]
+
```

Comment:
Note that we're adding the `Technical Preview` banner to the snmp-integration page only, and not to `input-snmp` and `input-snmptrap` pages.